xinference/client.py

```diff
@@ -76,6 +111,27 @@ def chat(
             Union["LlamaCppGenerateConfig", "PytorchGenerateConfig"]
         ] = None,
     ) -> Union["ChatCompletion", Iterator["ChatCompletionChunk"]]:
+        """
+        Given a list of messages comprising a conversation, the model will return a response.
+
+        Parameters
+        ----------
+        prompt : str
+            The user's input.
+        chat_history: Optional[List["ChatCompletionMessage"]]
+            A list of messages comprising the conversation so far.
+        generate_config: Optional["ChatglmCppGenerateConfig"]
```

Comment:
Incorrect type.

Also it is better to let users know the difference between LlamaCppGenerateConfig and PytorchGenerateConfig and which one to use.

Reply:
changed the incorrect type

And add model specified instructions